src/bikeracks/templates/scripts.js

```diff
@@ -575,14 +593,17 @@ BikeMap.prototype.arrowHTML = function(state, currentUserId) {
 
 
 BikeMap.prototype.popupContent = function(state) {
-    let onlineStatus;
+    let onlineStatus,
+        voterStatus;
     if (this.auth.currentUser) {
         onlineStatus = this.auth.currentUser.uid;
```

Comment:
Try to have `onlineStatus` only have one type. Bool? String? Right now this is actually `user_id` when the user is logged in and `user_logged_in` when they aren't logged in.

Reply:
Changed it to a bool.